recipe/build.sh

```diff
@@ -1,24 +1,25 @@
 #!/bin/env bash
+set -e
 
-
-# FIXME: This is a hack to make sure the environment is activated.
-# The reason this is required is due to the conda-build issue
-# mentioned below.
-#
-# https://github.com/conda/conda-build/issues/910
-#
-source activate "${CONDA_DEFAULT_ENV}"
-
-autoreconf --force --install
-./autogen.sh
-if [ "$(uname)" == "Linux" ]; then
-  export CXXFLAGS="${CXXFLAGS} -O2 -DNDEBUG"
+if [[ "$(uname)" == "Darwin" ]]; then
+    # at least for Travis's compiler, -O2 fails the unit tests
+    export CXXFLAGS="${CXXFLAGS} -O1"
 else
-  # There is at least on OSX a bug affecting -O2 builds
-  export CXXFLAGS="${CXXFLAGS} -O1 -DNDEBUG"
+    export CXXFLAGS="${CXXFLAGS} -O2"
 fi
+export CXXFLAGS="${CXXFLAGS} -DNDEBUG"
+
+mkdir build
+cd build
+cmake .. \
+    -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+    -DCMAKE_PREFIX_PATH="$PREFIX"
```

Comment:
Haven't tried, but [their `CMakeLists.txt`](https://github.com/google/snappy/blob/548501c98860bbe46e0eeda53942be67c7decef1/CMakeLists.txt#L82-L91) only defines `ADD_LIBRARY(snappy SHARED ...)`, with no mention of a static lib anywhere. https://github.com/google/snappy/pull/38/commits/ad817785de16d59c6bc6a0a3ca379ca145453d2c adds these options, sort of (it adds `BUILD_SHARED_LIBS` and builds the static libs if that's false).